stencils/pace/stencils/testing/translate.py

```diff
@@ -25,6 +31,17 @@ def pad_field_in_j(field, nj: int, backend: str):
     return outfield
 
 
+def get_data(value: Union[pace.util.Quantity, np.ndarray]) -> np.ndarray:
```

Comment:
nit: Can you name this `as_numpy` or similar?

Reply:
Sure!